### PR TITLE
Guide on unit testing

### PR DESCRIPTION
**Pull request by qunabu**
#### Description of what you did:

Documentation update on unit testing based on my medium post https://medium.com/qunabu-interactive/strapi-jest-testing-with-gitlab-ci-82ffe4c5715a. 

Sample repository to that includes the examples from the guide https://github.com/qunabu/strapi-unit-test-example

### CODE REVIEW

**Comment by derrickmehaffy** [comment]
Thank you for this guide! I've seen a number of users asking about this. :smiley:

**Comment by qunabu** [comment]
Code examples from the guide are here https://github.com/qunabu/strapi-unit-test-example is that helps.

**Comment by alexandrebodin** [comment]
Hello @qunabu thank you for this guide. We are releasing the stable version soon and the configuration changed. Your PR will not be up to date with the stable release. Can you look at the latest doc on the master branch and update your PR accordingly ?

**Comment by qunabu** [comment]
> Hello @qunabu thank you for this guide. We are releasing the stable version soon and the configuration changed. Your PR will not be up to date with the stable release. Can you look at the latest doc on the master branch and update your PR accordingly ?

Done

File: docs/3.0.0-beta.x/guides/unit-testing.md [hunk 1]
```diff
@@ -0,0 +1,312 @@
...
+```js
+const Strapi = require('strapi');
+const http = require('http');
+
+let instance;
+
+async function setupStrapi() {
+  if (!instance) {
+    /** the follwing code in copied from `./node_modules/strapi/lib/Strapi.js` */
+    await Strapi().load();
```
Comment (alexandrebodin) [inline docs/3.0.0-beta.x/guides/unit-testing.md:114]: we made some changes to the load function, it now calls the bootstrap directly

Comment (qunabu) [reply docs/3.0.0-beta.x/guides/unit-testing.md:114]: @alexandrebodin - Which of https://www.npmjs.com/package/strapi versions I should use in my example (https://github.com/qunabu/strapi-unit-test-example/blob/master/package.json) in order to use up to date version? 

I'm using latest `3.0.0-beta.20.3`

Comment (alexandrebodin) [reply docs/3.0.0-beta.x/guides/unit-testing.md:114]: You can use 3.0.0 :D

Comment (qunabu) [reply docs/3.0.0-beta.x/guides/unit-testing.md:114]: @alexandrebodin I've updated my example code https://github.com/qunabu/strapi-unit-test-example 

I'm going to update this as well, yet should this PR be into `master` or `documentation` branch?

**Comment by qunabu** [comment]
https://github.com/strapi/strapi/pull/6361/files that the improved version

**Pull request closed without merging**